azure/Guidelines.md

```diff
@@ -747,6 +749,20 @@ While removing a value from an enum is a breaking change, adding value to an enu
 
 > :ballot_box_with_check: **You SHOULD** use extensible enums unless you are positive that the symbol set will **NEVER** change over time.
 
+### Deprecating Behavior Notification
+
+When the [API Versioning](#API-Versioning) guidance above cannot be followed and the [Azure Breaking Change Reviewers](mailto:azbreakchangereview@microsoft.com) approve a breaking change to the operation/service, the operation/service that is being deprecated must add the `azure-deprecating` response header with a semicolon-delimited string notifying the caller what is being deprecated, when it will no longer function, and a URL linking to more information. The purpose is to inform customers (when debugging/logging responses) that they must take action to modify their call to the service's operation or their call will soon stop working entirely. It is not expected that client code will examine/parse this header's value in any way; it is purely informational to a human being.
```

Comment:
Do we want to include the breaking change reviewers email to the public doc? Also, recommend including a sentence about services using public communication channels/announcements to reach out to the customers (couldn't find it in the doc)

Reply:
We already had this email in the API Versioning section above so I just repeated it here. I agree that it's not ideal to have the email in the public doc but I think it's useful to have the reference in these guidelines. If we remove it, then it's harder for people to get in touch with us. I can say this: to date, this hasn't been a problem.

The notification mechanism is spelled out well in the Azure Breaking Change policy document and is somewhat involved/complicated - this we do not make public.